recipes/modlamp/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "modlamp" %}
+{% set version = "4.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "01a2035f4325419f73eb9a80bdcf4e0cca37940572ca7376778501dd5e1b08fc"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - lxml >=3.6.4
+    - matplotlib-base >=1.5.1
+    - nose >=1.3.7
+    - numpy >=1.10.4
+    - pandas >=0.18.1
+    - pip
+    - python
+    - requests >=2.11.1
+    - scikit-learn >=0.18.0
+    - scipy >=0.17.0
+    - setuptools >=20.2.2
+  run:
+    - lxml >=3.6.4
+    - matplotlib-base >=1.5.1
+    - nose >=1.3.7
+    - numpy >=1.10.4
+    - pandas >=0.18.1
+    - python
+    - requests >=2.11.1
+    - scikit-learn >=0.18.0
+    - scipy >=0.17.0
+    - setuptools >=20.2.2
+
+test:
+  imports:
+    - modlamp
+
+about:
+  home: "http://modlamp.org"
+  license: "BSD"
+  license_family: "BSD"
+  license_file: ""
```

Comment:
please add a license file here, if you have ...

Reply:
I removed the "licence_file" tag because the original repo has the license file.